backend/main.py

```
@@ -2,7 +2,20 @@
 from fastapi.responses import HTMLResponse
 from app.routers import auth, users, pose
 
-app = FastAPI()
+from contextlib import asynccontextmanager
+from app.database import engine, Base
+# Import all models to ensure they are registered with Base.metadata
+from app.models import user
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.14.8)

Unused function argument: `app`

(ARG001)